common/views/components/IIIFViewer/parts/GridViewer.js

```diff
@@ -103,27 +109,48 @@ const GridViewer = ({
   setActiveIndex,
   canvases,
   isFullscreen,
+  viewerRef,
 }: Props) => {
+  const { infoBanner } = useContext(GlobalAlertContext);
   const [newScrollOffset, setNewScrollOffset] = useState(0);
   const scrollVelocity = useScrollVelocity(newScrollOffset);
   const itemWidth = 250;
   const columnCount = Math.round(gridWidth / itemWidth);
   const columnWidth = gridWidth / columnCount;
   const grid = useRef();
-
   useEffect(() => {
     const rowIndex = Math.floor(activeIndex / columnCount);
     grid &&
       grid.current &&
       grid.current.scrollToItem({ align: 'start', rowIndex });
   }, [activeIndex]);
+  useEffect(() => {
+    setTimeout(() => {
+      window.scrollTo(0, 0);
+    }, 500);
+
+    const body = document && document.body ? document.body : null;
+    // there are multiple scrolls in this view, we have to set the body to hidden to stop flickering and offset
+    if (body && body.style) {
+      body.style.overflow = 'hidden';
```

Comment:
gridviewer at times feels like it needs to know more about state management of other components.

Reply:
Could you use `document?.body.?style.overflow = 'hidden`?

Reply:
I tried that but flow didnt like it .. i could add flowfixme and add the other way in.

Reply:
Na, we should adhere to the types as that's where we get safety. What was the error?

Reply:
document?.body?.style?.overflow = 'hidden';

Invalid left-hand side in assignment